modules/analysis/src/config.rs

```diff
@@ -0,0 +1,18 @@
+fn u64_from_size(s: &str) -> Result<u64, parse_size::Error> {
+    // keep it simple, allow only binary units (1024 based)
+    let cfg = parse_size::Config::new().with_binary();
+    cfg.parse_size(s)
+}
+
+#[derive(clap::Args, Debug, Clone)]
+pub struct AnalysisConfig {
+    #[arg(
+        id = "max-cache-size",
+        long,
+        env = "TRUSTD_MAX_CACHE_SIZE",
```

Comment:
super!

Reply:
nit mind to add a line on https://github.com/trustification/trustify/blob/main/docs/env-vars.md ?

Reply:
will do